core/feconf.py

```diff
@@ -922,6 +922,7 @@ def get_empty_ratings() -> Dict[str, int]:
 CONTRIBUTOR_ALL_STATS_SUMMARIES_URL = '/contributorallstatssummaries'
 CONTRIBUTOR_CERTIFICATE_URL = '/contributorcertificate'
 CONTRIBUTOR_DASHBOARD_ADMIN_URL = '/contributor-dashboard-admin'
+CONTRIBUTOR_DASHBOARD_ADMIN_STATS_URL = '/contributor-dashboard-admin-stats'
```

Comment:
Can you please once confirm the naming convention we use here, at some places we separate the keywords by a hyphen(-), at some places by an underscore(_), and at some places its continuous text, maybe with @seanlip?
Just want to be sure so that we follow a similar pattern.
Thanks

Reply:
Hi @seanlip , does the convention used correct? Or it should be something like hitesh suggested?

Reply:
User-facing URLs (that are shown in the URL bar) should be hyphenated. Data handlers can be concatenated (but hyphenated is probably OK too).